python-package/lightgbm/basic.py

```diff
@@ -196,6 +196,48 @@ class LightGBMError(Exception):
                      "feature_penalty": C_API_DTYPE_FLOAT64,
                      "monotone_constraints": C_API_DTYPE_INT8}
 
+CONFIG_ALIASES = {"boosting": {"boosting",
```

Comment:
Can we please turn this into a class? class params

Reply:
Sorry, not sure that I understood you correctly. Please take a look.